plugins/grafana-custom-plugins/grafana-http-table-plugin/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "grafana-table-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
+    "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "test": "jest --silent",
+    "test:ci": "jest --passWithNoTests --maxWorkers 4",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
+    "lint:fix": "npm run lint -- --fix",
+    "e2e": "npm exec cypress install && npm exec grafana-e2e run",
```

Comment:
Remove unused scripts and dependencies, which might create unwanted security issue alert after it is merged.